docs/deploying.md

```diff
@@ -369,8 +372,159 @@ support.
 		* Closing connection 0
 		curl: (60) SSL certificate problem: self signed certificate
 		More details here: http://curl.haxx.se/docs/sslcerts.html
+
+## Deploy a v2 registry with Nginx and basic authentication
+
+This example shows you how to securely deploy a v2 registry using basic authentication.
+
+### Install Docker Compose
+
+1. Open a new terminal on the host with your `distribution` directory.
+
+2. Get the `docker-compose` binary.
+
+		$ sudo wget https://github.com/docker/compose/releases/download/1.3.0/docker-compose-`uname  -s`-`uname -m` -O /usr/local/bin/docker-compose
+
+	This command installs the binary in the `/usr/local/bin` directory. 
+	
+3. Add executable permissions to the binary.
+
+		$  sudo chmod +x /usr/local/bin/docker-compose
+
+### Do some housekeeping
```

Comment:
This step seems distracting. Is it completely necessary to get through the build? People will often run these commands without even reading what they are for. This might belong in a note.